### ISSUE TITLE
BTTV Chat settings aren't showing up in the settings anymore.

### ISSUE THREAD

**Issue by MisterQuiggly** [night/BetterTTV#2774]
When I click the gear below the area where I can type my messages, the BTTV settings do not show up anymore. They used to but now they don't.

**Comment by SaiyanOfDarkness** [comment]
Yeah mine either, it said something about "stuff will be broken temporarily, be patient" or something..

**Comment by GhostStalker88** [comment]
It appears that you no longer go to the gear area for BTTV settings. You can now access them under the drop down menu when you click on your name in the top right of the screen. There is a entry of "BetterTTV Settings" right above the logout option.

**Comment by night** [comment]
fix for this coming soon
